### DISCUSSION TITLE
【0x03 Project】尝试运行scaffold-eth！

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#7] [Projects]
在前面几个 Project 中，我们通过 `eth.build` 来了解区块链基本原理。
从 `Project 0x03` 开始，我们开始正式接触 dApp 了，第一个任务，把 Scaffold-Eth 脚手架运行起来！

教程：

https://mp.weixin.qq.com/s/72gzBaLNA0ruHSPsVBPtmQ

环境配置参考—— `nvm` 与 `node.js` 安装配置教程：
```
curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.2/install.sh | bash

export NVM_DIR="$HOME/.nvm"
[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" # This loads nvm

# nvm换源：
export NVM_NODEJS_ORG_MIRROR=https://npm.taobao.org/mirrors/node/

# 可能问题解决：
sudo apt-get install build-essential

# npm 和 yarn换源：
npm config set registry https://registry.npm.taobao.org

yarn config set registry https://registry.npm.taobao.org/
```

**Comment by lysion** [comment]
<img width="1196" alt="Screen Shot 2021-11-18 at 10 21 42 PM" src="https://user-images.githubusercontent.com/30424467/142433748-45ffac1c-2e6b-4ffa-beff-283505173f5c.png">

**Comment by leeduckgo** [comment]
Cool!

**Comment by gofenix** [comment]
![image](https://user-images.githubusercontent.com/6822558/147917315-6aa6beee-958c-4c78-a6df-f25f10b4ccbc.png)

**Comment by xFANx** [comment]
![image](https://user-images.githubusercontent.com/45059192/161447572-d6cf3ef2-f8c0-4853-aa8c-3716489b5bef.png)
